content/Lab1/20_Deploy_FTD_FMC/index.md

```diff
@@ -1,20 +1,20 @@
 ---
 title: "Deploy FTD and FMC in AWS"
-weight: 40
+weight: 20
 ---
 
 In this section, we will deploy Cisco FTDv and FMCv in AWS using terraform automation. We will first describe and walk through the relevant terraform code snippets and then proceed with the deployment.
 
 If you are using Event Engine:
 
-[Cisco Secure Firewall Setup - Description](../40_Deploy_FTD_FMC/1_FTD_and_FMC_Event_Engine.md)
+[Cisco Secure Firewall Setup - Description](../20_Deploy_FTD_FMC/1_FTD_and_FMC_Event_Engine.md)
 
 ---
 
 If you are using AWS Account:
 
-[Cisco Secure Firewall Setup - Description](../40_Deploy_FTD_FMC/2_FTD_and_FMC_AWS_Account.md)
+[Cisco Secure Firewall Setup - Description](../20_Deploy_FTD_FMC/2_FTD_and_FMC_AWS_Account.md)
 
 ---
 
-[Cisco Secure Firewall - Deployment](../40_Deploy_FTD_FMC/3_Deployment.md)
\ No newline at end of file
+[Cisco Secure Firewall - Deployment](../20_Deploy_FTD_FMC/3_Deployment.md)
```

Comment:
please rename the respective md titles so that it matches with this text e.g. 1_network_resources.md title should be AWS Network Resources - Description